.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
as far as I know this is not valid JSON. Is VSCode OK with it?

Reply:
More than OK, VSCode generated it :-D

Reply:
And, well, it _should_ be valid json. Mr Crockford was wrong on this.